PSTenable/Public/Get-PSTenableSeverity.ps1

```diff
@@ -26,66 +26,93 @@ function Get-PSTenableSeverity {
             'Medium',
             'Low'
         )]
-        [string]
-        $Severity
+        [string[]]
+        $Severity,
+        
+        [Parameter(Position = 0, Mandatory = $false)]
+        [int]$maxrecords = 0

```

Comment:
added so caller can specify max records (at which the query will just stop asking SC)